scripts/pvscheck.sh

```diff
@@ -263,16 +263,29 @@ create_compile_commands() {(
 
   if test -z "$deps" ; then
     mkdir -p "$tgt/build"
-    cd "$tgt/build"
+    (
+      cd "$tgt/build"
 
-    cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX="$PWD/root"
-    make -j"$(get_jobs_num)"
+      cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX="$PWD/root"
+      make -j"$(get_jobs_num)"
+    )
   else
-    cd "$tgt"
+    (
+      cd "$tgt"
 
-    make -j"$(get_jobs_num)" CMAKE_EXTRA_FLAGS=" -DCMAKE_INSTALL_PREFIX=$PWD/root -DCMAKE_BUILD_TYPE=Debug "
+      make -j"$(get_jobs_num)" CMAKE_EXTRA_FLAGS=" -DCMAKE_INSTALL_PREFIX=$PWD/root -DCMAKE_BUILD_TYPE=Debug "
+    )
   fi
-  find src/nvim/auto -name '*.test-include.c' -delete
+  find "$tgt/build/src/nvim/auto" -name '*.test-include.c' -delete
+)}
+
+# Warning: realdir below only cares about directories unlike realpath.
+#
+# realpath is not available in Ubuntu trusty yet.
```

Comment:
Good to know. Gentoo still has realpath package and I remember installing it, so I did not think something changed. In the current state it appears this package is for something else (FreeBSD alternative to coreutils?) and is actually not installed on my system, though realpath is there.